src/Modules/Profile/Commands.Profile/help/Invoke-PowerBIRestMethod.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.PowerBI.Commands.Profile.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Profile
+online version:
+schema: 2.0.0
+---
+
+# Invoke-PowerBIRestMethod
+
+## SYNOPSIS
+Executes a REST call to the Power BI service with provided URL and body.
+
+## SYNTAX
+
+```
+Invoke-PowerBIRestMethod -Url <String> -Method <PowerBIWebRequestMethod> [-Body <String>]
+ [-Organization <String>] [-Version <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Invokes a REST request against the Power BI service using your logged in profile (Connect-PowerBIServiceAccount).
+The type of REST question is determined by -Method and body is required for certain methods such as POST, PUT, and PATCH.
```

Comment:
Consider revising for clarity.
Example: "The REST verb can be specified using the -Method parameter. -Body is required for certain methods such as POST, PUT, and PATCH."